src/vehicle.cpp

```diff
@@ -41,6 +41,13 @@
 #include <numeric>
 #include <algorithm>
 
+/**
+ *  Constant use to convert between wheel and engine rpm
+ *  1:3.7 is a fairly typical differential gearing
+ *  0.15 is a magic constant based upon ~15" tyres
+ */
+static constexpr float wheel_diff = 3.7 * 0.15;
```

Comment:
Now that certainly sounds like the kind of detail we really, really don't need to be reflecting in the game.
It's pure magic - being based on real life constants doesn't help one bit here.